src/App.js

```diff
@@ -0,0 +1,70 @@
+import logo from './logo.svg'
+import './App.css'
+
+function App() {
+  return (
+    <>
+      <section className='section'>
+        <nav className='nav'>
```

Comment:
this is not react; :cry:  you should create components  from navbar to hero to cards with info